docs/EN_US/ECLLanguageReference/ECLR_mods/ExpressionsandOperatos.xml

```diff
@@ -80,7 +80,17 @@
       </tgroup>
     </informaltable>
 
-    <para>Division by zero<indexterm>
+    <para><variablelist>
+        <varlistentry>
+          <term>Note:</term>
+
+          <listitem>
+            <para>* , /, %, and DIV all have the same precedence and are left
```

Comment:
There is still a comment:
" are supported for use in expressions, listed here in their evaluation precedence" at the head of the table,
which is likely to be confusing.